README.md

```diff
@@ -40,7 +39,7 @@ Rem: The folder [your-local-project-folder]/SampleCommunicator/src/main/resource
 <table valign="top">
     <tr>
         <td><b>Implementor:</b></td>
-        <td><p>Communicator Provider (3rd Party)</p></td>
+        <td><p>+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++Communicator Provider (3rd Party)</p></td>
```

Comment:
@ergo-furrer This look very strange!

Reply:
This was not intended. Not sure how this happened...

Reply:
@ssabatini seems that I slept on the keyboard for a while 🙄 ...

Reply:
:-)